src/main/java/org/embulk/util/config/ConfigMapper.java

```diff
@@ -81,16 +79,12 @@
         final T value;
         try {
             value = this.objectMapper.readValue(objectNode.traverse(), taskType);
-        } catch (final JsonMappingException ex) {
-            throw new ConfigException("Failed to map a JSON value into some object.", ex);
-        } catch (final JsonParseException ex) {
-            throw new ConfigException("Unexpected failure in parsing ObjectNode rebuilt from org.embulk.config.ConfigSource.", ex);
         } catch (final JsonProcessingException ex) {
-            throw new ConfigException("Unexpected failure in processing ObjectNode rebuilt from org.embulk.config.ConfigSource.", ex);
+            throw new ConfigException(buildExceptionMessage(ex, taskType), ex);
```

Comment:
these code blocks are similiar, could we `catch JsonProcessingException | IOException | RuntimeException` ?

Reply:
Reasonable suggestion. Addressed this in 4833c81c30484a820df2c52764f6a86c9b6497f2.

(As `JsonProcessingException` is a sub class of `IOException`, they're merged.)